### ISSUE TITLE
Support for record?

### ISSUE THREAD

**Issue by ziaulhasanhamim** [DamianEdwards/MiniValidation#31]
Most of the time I use records as my request, and response objects. But Validation doesn't work with them.
```c#
public record Request([Required] string Text);
```
If I want it to work I need to add the extra keyword 'property'
```c#
public record Request([property:Required] string Text);
```
Isn't that redundant? In MVC it works without that.

**Comment by DamianEdwards** [comment]
Without the `property:` prefix, the attribute actually applies to the constructor parameter. I didn't actually realize that MVC supports this now. I'll take a look at adding support for this pattern as I agree it's nice.

**Comment by DamianEdwards** [comment]
Fixed in 065c6efb4214db2fe5400f962b9da20001f04fb9
